packages/ros2/src/SystemHandle.cpp

```diff
@@ -25,6 +25,17 @@
 
 #include <rclcpp/executors/single_threaded_executor.hpp>
 
+#ifdef WIN32
+#define SETENV(id,value,b,retValue) \
+	std::ostringstream _aux_d; \
+	_aux_d << id << "=" << value; \
+	retValue += _putenv(_aux_d.str().c_str());
+#define UNSETENV(id,retValue) SETENV(id,"",false,retValue)
```

Comment:
A "tikismikis" change: 
```suggestion
#define UNSETENV(id,retValue) SETENV(id, "", false, retValue)
```

Reply:
Ok, Done ;)